src/treetensornetworks/opsum_to_ttn.jl

```diff
@@ -236,34 +223,61 @@ function ttn_svd(
 
       push!(tempTTN[v], el)
     end
+    remove_dups!(tempTTN[v])
+  end
+
+  return tempTTN, inbond_coefs
+end
 
-    ITensorMPS.remove_dups!(tempTTN[v])
-    # manual truncation: isometry on incoming edge
+function svd_bond_coefs(
+  coefficient_type, sites, ordered_verts, ordered_edges, inbond_coefs; kws...
+)
+  Vs = Dict(e => Dict{QN,Matrix{coefficient_type}}() for e in ordered_edges)
```

Comment:
This could be a `DataGraph` with that data on the edges of the graph.

I also wonder if `Dict{QN,Matrix{coefficient_type}}` could be a block diagonal `BlockSparseMatrix` where those matrices are the diagonal blocks and the QNs are the sector labels of the graded axes.

Reply:
Using `DataGraph`s could be really helpful. There are a lot of complicated dictionaries where the keys are things like edge=>QN pairs, which makes logical sense and is a carryover from the MPO code, but still looks too complicated. So maybe a `DataGraph` with a dictionary having QN keys on each edge could work and would be much cleaner.